contrib/net/http/roundtripper_test.go

```diff
@@ -88,6 +89,12 @@ func TestRoundTripper(t *testing.T) {
 	assert.Equal(t, true, s1.Tag("CalledAfter"))
 	assert.Equal(t, ext.SpanKindClient, s1.Tag(ext.SpanKind))
 	assert.Equal(t, "net/http", s1.Tag(ext.Component))
+	assert.Equal(t, "127.0.0.1", s1.Tag(ext.NetworkDestinationName))
+
+	wantPort, err := strconv.Atoi(strings.TrimPrefix(s.URL, "http://127.0.0.1:"))
+	require.NoError(t, err)
+	require.NotEmpty(t, wantPort)
+	assert.Equal(t, wantPort, s1.Tag(ext.NetworkDestinationPort))
```

Comment:
what's the expected value here? 0 I guess? Is it a valid value?
And for better readability can we just put the numeric value directly instead of `strconv.Atoi(strings.TrimPrefix(s.URL, "http://127.0.0.1:"))` ?